pkg/skaffold/runner/labeller.go

```diff
@@ -51,3 +51,7 @@ func (d *DefaultLabeller) Labels() map[string]string {
 		K8ManagedByLabel: fmt.Sprintf("skaffold-%s", version),
 	}
 }
+
+func (d *DefaultLabeller) K8sMangedByLabel() string {
+	return fmt.Sprintf("%s=skaffold-%s", K8ManagedByLabel, d.version)
```

Comment:
This took me a while to understand - can we rename/refactor some things to make it clearer? 
*  `K8ManagedByLabel`  -> `K8sManagedByLabelKey` 
* `K8sMangedByLabel()` -> `K8sManagedByLabelKeyValueString()`
* `fmt.Sprintf("skaffold-%s", version)` -> `skaffoldVersion()` and use that in `K8sManagedByLabelKeyValueString()`

Reply:
for sure, done!